src/Validator/RegExp.php

```diff
@@ -16,6 +16,7 @@ public function __construct(
    * @throws \UnexpectedValueException
    */
   public function __invoke(mixed $value): string {
+    /** @psalm-suppress ArgumentTypeCoercion */
```

Comment:
@Chi-teck Just a heads up, I had to suppress an error from psalm here. Let me know if you had a different solution in mind.